docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -1993,6 +1999,16 @@ definitions:
           'user_defined'.
         type: string
         example: '0'
+      partition_compaction_key:
+        description: |
+          Represents a key of event for per-partition compaction of this event type.
```

Comment:
What do you think about "Value used for per-partition compaction of the event type. Given two events with the same `partition_compaction_key` value are published to the same partition, the later overwrites the former.". It's a kind of redundant but I suggest it because it would be self-contained information about what it means.

Reply:
I like it. Will change.

Reply:
Fixed https://github.com/zalando/nakadi/pull/905/commits/29a4aaba4702bd72f4f45ad4821eca626b1a2d87